README.md

```diff
@@ -385,28 +383,28 @@ Tags can be in two forms:
 
 Both can coexist with standard Tag parsing.
 
-Tag                    | Description
------------------------| -------------------------------------------
-`cmd`                  | If present, struct is a command.
-`arg`                  | If present, field is an argument.
-`env:"X"`              | Specify envar to use for default value.
-`name:"X"`             | Long name, for overriding field name.
-`help:"X"`             | Help text.
-`type:"X"`             | Specify [named types](#custom-named-types) to use.
-`placeholder:"X"`      | Placeholder text.
-`default:"X"`          | Default value.
-`short:"X"`            | Short name, if flag.
-`required`             | If present, flag/arg is required.
-`optional`             | If present, flag/arg is optional.
-`hidden`               | If present, command or flag is hidden.
-`format:"X"`           | Format for parsing input, if supported.
-`sep:"X"`              | Separator for sequences (defaults to ","). May be `none` to disable splitting.
-`enum:"X,Y,..."`       | Set of valid values allowed for this flag.
-`group:"X"`            | Logical group for a flag or command.
-`xor:"X"`              | Exclusive OR group for flags. Only one flag in the group can be used which is restricted within the same command.
-`prefix:"X"`           | Prefix for all sub-flags.
-`set:"K=V"`            | Set a variable for expansion by child elements. Multiples can occur.
-`embed`                | If present, this field's children will be embedded in the parent. Useful for composition.
+| Tag               | Description                                                                                                       |
```

Comment:
Mind reverting this bit? I've found that having the extra `|`s makes it just that bit more annoying to maintain these tables.

Reply:
oups, very sorry. That was prettier doing it's markdown formatting.
Done.